docs/src/extend/custom-rule-plugin-tutorial.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Custom Rule & Plugin Tutorial
```

Comment:
I think we may just want to call this "Custom Rule Tutorial". I re-read this a couple of times and I kept thinking it reads like two separate tutorials will be in this page. Logically, creating a plugin is the latest step in creating a custom rule, so I think we're okay with shortening this title.
```suggestion
title: Custom Rule Tutorial
```

Reply:
will also update the page name